server/server.py

```diff
@@ -24,34 +24,37 @@ def handle(self):
 		try:
 			msg = self.RSAcipher.decrypt(data).decode().split(" ", 1)
 
-			if (msg[0] == '3317BLT5'):
+			if (msg[0] != '3317BLT5'):
+				raise BaseException("Invalid magic number")
```

Comment:
That is an interesting thought. As it stands all of our logging outside of proper check-ins are just stdout for reasons stated below. If we ever implement some sort of stderr we would need to look into if we need to add a write here or in the except. personally if we write to stderr in the except then it can handle line 25 and 28 errors in one place.